Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM mysocietyorg/apache-php-fpm:stretch
+LABEL maintainer="sysadmin@mysociety.org"
+
+RUN apt-get update \
+      && apt-get install -y \
+          libconvert-base32-perl \
```

Comment:
Can a Dockerfile not read a list from a file somehow? :) COPY conf/packages inside and then run it on that?

Reply:
Yes, we could, as long as the packages file is a simple list of package, we could `COPY` it in then do something like `RUN xargs -a packages apt-get install -y --no-install-recommends`.

Reply:
Added in [3846d20](https://github.com/mysociety/writetothem/pull/464/commits/3846d2098719cfdefec380eb386b27d225a2dc01)